api/src/routes/settings.test.ts

```diff
@@ -689,6 +689,29 @@ Please wait 5 minutes to resend an authentication link.`
         expect(response.statusCode).toEqual(400);
       });
     });
+
+    describe('/update-my-classroom-mode', () => {
+      test('PUT returns 200 status code with "success" message', async () => {
+        const response = await superPut('/update-my-classroom-mode').send({
+          isClassroomAccount: true
+        });
+
+        expect(response.body).toEqual({
+          message: 'flash.classroom-mode-updated',
+          type: 'success'
+        });
+        expect(response.statusCode).toEqual(200);
+      });
+
+      test('PUT returns 400 status code with invalid classroomMode', async () => {
```

Comment:
Not sure if this really matters, but the current API sends a 403 back when there isn't a valid request body.